utilities_common/cli.py

```diff
@@ -127,7 +127,6 @@ def __init__(self, db=None):
 
 
         if not self.port_dict:
-            click.echo(message="Warning: failed to retrieve PORT table from ConfigDB!", err=True)
```

Comment:
>, err=True [](http://example.com/codeflow?start=86&length=10)

This is a good warning message. Can you keep it and just remove `, err=True` ?

Reply:
@qiluo-msft 
I think that if we are going to leave this message, we should at least change it, something like "Configuration database contains no ports" and of course remove err=True.
What do you think?

Reply:
Agree.

Reply:
Done.